File: src/main/java/com/lycoon/clashapi/old_models/clan/Location.java
package com.lycoon.clashapi.old_models.clan;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Location
{
	@SerializedName("id")
	@Expose
	private Integer id;
	
	@SerializedName("name")
	@Expose
	private String name;
	
	@SerializedName("isCountry")
	@Expose
	private Boolean isCountry;
	
	@SerializedName("countryCode")
	@Expose
	private String countryCode;

	public Integer getId()
	{
		return id;
	}

	public String getName()
	{
		return name;
	}

	public Boolean getIsCountry()
	{
		return isCountry;
	}

	public String getCountryCode()
	{
		return countryCode;
	}
}

File: src/main/java/com/lycoon/clashapi/models/common/Location.kt
package com.lycoon.clashapi.models.common

import kotlinx.serialization.Serializable

@Serializable
data class Location(
    val localizedName: String? = null,
    val id: Int = 0,
    val name: String? = null,
    val isCountry: Boolean = false,
    val countryCode: String? = null // if isCountry false
)